.markdownlint.yml

```
@@ -1,16 +1,22 @@
 # Default state for all rules
-default: true
+default: false
 MD003:
   # Heading style
   style: 'atx'
-MD004: false
-# Can't disable MD007 :/
-MD007: false
-MD009: false
-MD010:
-  code_blocks: false
-MD013: false
-MD024: false
-MD029: false
-MD033: false
-MD037: false # breaks on latex
+MD018: true
+MD026: true
+MD034: true
+MD040: true
+MD042: true
+MD048: true
```

Link fragments should be valid

```suggestion
MD048: true
MD051: true
```